e2e/testdata/test-service/main.go

```diff
@@ -136,7 +147,7 @@ func main() {
 		}
 
 		if _, err := client.EventClient.Create(context.Background(), &pb.CreateEventRequest{
-			InstanceHash: exec.InstanceHash,
+			InstanceHash: client.InstanceHash,
```

Comment:
Why change to client? It was here to make sure Execution has proper hash.

Reply:
The event should not be linked with the execution.
Both event and execution should use the hash from ENV